### DISCUSSION TITLE
Logging knobs

### DISCUSSION THREAD

**Discussion by jhammond-git** [bb-Ricardo/netbox-sync#166] [Ideas]
I see that there's a settings section for some knobs on logging.  Are there any for things like log rotation and retention or would I have to dig into one of the python files to find that?

**Comment by jhammond-git** [comment]
I finally dug into the files and discovered that rotation is default, I just think I haven't hit the max file size yet to trigger it, as I've blanked out my log file several times to help manage debugging issues.

**Reply by bb-Ricardo** [reply]
The script is using the default python tools for logging. I don't really know how to help you here.

What do you try to achive? Smaller size for log rotation?

I could add a config value to set the size limit and logfiles to keep.

**Reply by jhammond-git** [reply]
No worries.  I just hadn't dug into where the logging was set up.  Since the max file size is 10MB, I hadn't run into the limit yet.
